api/professor.py

```diff
@@ -98,3 +99,40 @@ def delete_professor(professor_id):
         return f'Unable to delete prof with id {professor_id}', 500
 
     return f'Deleted prof with id {professor_id}', 200
+
+@PROFESSOR_BP.route('/remind', methods=['POST'])
```

Comment:
We could move the uuid into the url instead of passing through json, this would align with our other endpoints where we pass uuid through url. 
If you decide to go through with the suggested change then you can move remind.yaml contents into professors_{id}.yaml because it would be a subset of that path. 

Everything else looks good!

Reply:
Hi Annie, my thinking was that unlike those other urls, this one is a URL to perform an action rather than to get something like the other ones. When I was doing some research it suggested that URLs that perform actions should not be GET endpoints as GET should mainly be used for retrieving things.

Reply:
Yup this definitely makes sense as a POST endpoint, not a GET endpoint. We do have POST endpoints that take uuid through the url as well. However, I'm not aware of any issues there may be with taking uuid through url. It is somewhat easier to pass uuid through URL than JSON but then that leaves the POST endpoint without a payload which feels a bit odd as well.